### PR TITLE
fix(RHTAPWATCH-1140): Remove segment-bridge components from infra-deployments

### PR DESCRIPTION
**Pull request by Kousalya1998**
- Remove mentions of segment-bridge and its components

### CODE REVIEW

**Review by amisstea — APPROVED** [review]
@Kousalya1998 the changes here look ok. Since this will remove access to the clusters for the jobs running in MP+, please sure those cronjobs are disabled or removed prior to merging this.

**Review by hmariset — APPROVED** [review]
LGTM

**Review by ifireball — APPROVED** [review]
Approved, with the caveat @amisstea already mentioned

**Comment by Kousalya1998** [comment]
> @Kousalya1998 the changes here look ok. Since this will remove access to the clusters for the jobs running in MP+, please sure those cronjobs are disabled or removed prior to merging this.

Working on it. Created a SNOW request and its in process
https://redhat-internal.slack.com/archives/C04LZLPE7QU/p1721312613998879

**Comment by Kousalya1998** [comment]
@simonbaird need your final approval to merge this PR

**Comment by simonbaird** [comment]
> @simonbaird need your final approval to merge this PR

I don't know much about segment-bridge, are you sure I'm the person you need?

File: argo-cd-apps/base/member/infra-deployments/kustomization.yaml [hunk 1]
```diff
@@ -24,4 +23,4 @@ resources:
   - tracing-workload-otel-collector
   - tempo
 components:
-  - ../../../k-components/inject-infra-deployments-repo-details
+  - ../../../k-components/inject-infra-deployments-repo-details
```
Comment (simonbaird) [inline argo-cd-apps/base/member/infra-deployments/kustomization.yaml:26]: Fix the whitespace error maybe?

Comment (Kousalya1998) [reply argo-cd-apps/base/member/infra-deployments/kustomization.yaml:26]: Sure, on it

**Review by simonbaird — APPROVED** [review]
Token approve since I'm on the reviewer list, but beware I have no clue about this patch! If it breaks everything, I won't be responsible! :wink:

**Comment by amisstea** [comment]
@simonbaird unfortunately none of the OWNERS members are SMEs. @ifireball is the primary SME for segment-bridge and our team, o11y, maintains it. Does that help enough to get a /lgtm? :smile:

**Comment by ifireball** [comment]
@amisstea @Kousalya1998 were the jobs in MPaaS+ removed?

**Review by mmorhun — APPROVED** [review]

**Comment by simonbaird** [comment]
Perhaps the failures [here](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/redhat-appstudio_infra-deployments/4072/pull-ci-redhat-appstudio-infra-deployments-main-appstudio-hac-e2e-tests/18186369448007475200) need some attention..?

**Review by ralphbean — APPROVED** [review]

**Pull request merged**